rust_grpcio_bench/src/main.rs

```diff
@@ -21,7 +21,7 @@ struct GreeterService;
 impl Greeter for GreeterService {
     fn say_hello(&mut self, ctx: RpcContext<'_>, req: HelloRequest, sink: UnarySink<HelloReply>) {
         let mut resp = HelloReply::default();
-        resp.set_message(req.get_name().to_string());
+        resp.set_response(req.get_request().clone());
```

Comment:
could we move the request instead of cloning? given that we remove debug print of `req` from error handling later on?

Reply:
hint: `take_request()`